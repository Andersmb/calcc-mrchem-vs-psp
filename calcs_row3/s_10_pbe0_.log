Starting job 5688134 on c1-[21,26],c3-[13,27],c5-22,c10-5 at Fri May 20 06:39:38 CEST 2022

Copying s_10_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying s_10_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688134      s_10_pbe0_        180                                             01:54:40      0:0 
5688134.bat+      batch         24        1   00:00:08          0   00:00:08   01:54:40      0:0 
5688134.ext+     extern        180        6   00:00:00          5   00:00:00   01:54:40      0:0 
5688134.0      mrchem.x        180       30   01:53:24         29   05:54:35   01:53:38      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688134                                                                          
5688134.bat+     19340K          0     19340K        0              0          0 
5688134.ext+          0          5          0        0              5          0 
5688134.0     44277500K         29 138920531+        0              9          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688134                                                                                               
5688134.bat+       19.45M               0         19.45M        0.08M                0          0.08M 
5688134.ext+        0.00M               5          0.00M            0                5              0 
5688134.0           7.32M              27          6.81M      317.94M                0        239.17M 

Job 5688134 completed at Fri May 20 08:34:19 CEST 2022
