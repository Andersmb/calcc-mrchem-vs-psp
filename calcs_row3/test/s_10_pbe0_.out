                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :        0.000000        0.000000        0.000000
    1       F            :        0.000000        0.000000        2.949295
    2       F            :       -2.085467        2.085467        0.000000
    3       F            :        2.085467       -2.085467        0.000000
    4       F            :        2.085467        2.085467        0.000000
    5       F            :       -2.085467       -2.085467        0.000000
    6       F            :        0.000000        0.000000       -2.949295
---------------------------------------------------------------------------
 Center of mass          :       -0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              35
 Beta electrons          :                                              35
 Total electrons         :                                              70
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000002117e+00
    1    1    a          :                              9.999986311897e-01
    2    1    a          :                              9.999993455948e-01
    3    1    a          :                              9.999999910795e-01
    4    1    a          :                              9.999999842799e-01
    5    1    a          :                              9.999999946418e-01
    6    1    a          :                              1.000001376174e+00
    7    1    a          :                              9.999999976352e-01
    8    1    a          :                              9.999999873752e-01
    9    1    a          :                              9.999999856898e-01
   10    1    a          :                              9.999999970374e-01
   11    1    a          :                              9.999999017540e-01
   12    1    a          :                              9.999999569430e-01
   13    1    a          :                              9.999999776399e-01
   14    1    a          :                              9.999998391454e-01
   15    1    a          :                              1.000000025302e+00
   16    1    a          :                              1.000000287057e+00
   17    1    a          :                              1.000000940366e+00
   18    1    a          :                              1.000000211102e+00
   19    1    a          :                              9.999981963379e-01
   20    1    a          :                              9.999993881265e-01
   21    1    a          :                              1.000000229105e+00
   22    1    a          :                              1.000000184413e+00
   23    1    a          :                              9.999971287846e-01
   24    1    a          :                              9.999999777750e-01
   25    1    a          :                              9.999969099089e-01
   26    1    a          :                              9.999998145771e-01
   27    1    a          :                              9.999991590332e-01
   28    1    a          :                              1.000003315720e+00
   29    1    a          :                              9.999993553626e-01
   30    1    a          :                              9.999996831549e-01
   31    1    a          :                              1.000000316348e+00
   32    1    a          :                              1.000000165446e+00
   33    1    a          :                              1.000000176693e+00
   34    1    a          :                              1.000003165039e+00
   35    1    b          :                              1.000000002117e+00
   36    1    b          :                              1.000004317469e+00
   37    1    b          :                              9.999997944814e-01
   38    1    b          :                              1.000000082632e+00
   39    1    b          :                              1.000000125856e+00
   40    1    b          :                              1.000000747608e+00
   41    1    b          :                              1.000001390381e+00
   42    1    b          :                              9.999999977576e-01
   43    1    b          :                              9.999999872911e-01
   44    1    b          :                              9.999999843127e-01
   45    1    b          :                              9.999999970371e-01
   46    1    b          :                              9.999997475286e-01
   47    1    b          :                              9.999999659912e-01
   48    1    b          :                              9.999999824052e-01
   49    1    b          :                              9.999998527606e-01
   50    1    b          :                              1.000000099849e+00
   51    1    b          :                              1.000000078986e+00
   52    1    b          :                              1.000001610714e+00
   53    1    b          :                              1.000000212336e+00
   54    1    b          :                              9.999981964401e-01
   55    1    b          :                              9.999993128388e-01
   56    1    b          :                              1.000000229103e+00
   57    1    b          :                              9.999999887407e-01
   58    1    b          :                              1.000000173717e+00
   59    1    b          :                              9.999999771700e-01
   60    1    b          :                              9.999977464400e-01
   61    1    b          :                              9.999997303845e-01
   62    1    b          :                              9.999988535196e-01
   63    1    b          :                              1.000003043341e+00
   64    1    b          :                              9.999993574280e-01
   65    1    b          :                              9.999996346831e-01
   66    1    b          :                              1.000001979380e+00
   67    1    b          :                              9.999999361333e-01
   68    1    b          :                              1.000001161374e+00
   69    1    b          :                              1.000004165854e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                986.659707177948
 E-N energy              :            (au)              -3486.801077025889
 Coulomb energy          :            (au)               1030.484787645000
 Exchange energy         :            (au)                -21.487109157992
 X-C energy              :            (au)                -66.798470211288
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                567.188970147096
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1557.942161572222
 Nuclear energy          :            (au)                567.188970147096
---------------------------------------------------------------------------
 Total energy            :            (au)             -9.907531914251e+02
                         :      (kcal/mol)             -6.217070090612e+05
                         :        (kJ/mol)             -2.601222127600e+06
                         :            (eV)             -2.695976658139e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -89.485119173697
    1    1    a          :            (au)                -24.652434744438
    2    1    a          :            (au)                -24.651449356083
    3    1    a          :            (au)                -24.599846798486
    4    1    a          :            (au)                -24.598882899946
    5    1    a          :            (au)                -24.549178717762
    6    1    a          :            (au)                -24.545512184462
    7    1    a          :            (au)                 -8.858879166580
    8    1    a          :            (au)                 -6.810250552584
    9    1    a          :            (au)                 -6.809822616722
   10    1    a          :            (au)                 -6.809099694843
   11    1    a          :            (au)                 -1.577255622793
   12    1    a          :            (au)                 -1.397561399639
   13    1    a          :            (au)                 -1.376108528782
   14    1    a          :            (au)                 -1.350047234439
   15    1    a          :            (au)                 -1.285447075994
   16    1    a          :            (au)                 -1.256169485714
   17    1    a          :            (au)                 -0.845522169104
   18    1    a          :            (au)                 -0.713463063835
   19    1    a          :            (au)                 -0.696093543657
   20    1    a          :            (au)                 -0.673047518187
   21    1    a          :            (au)                 -0.552405200997
   22    1    a          :            (au)                 -0.543517878614
   23    1    a          :            (au)                 -0.528382156713
   24    1    a          :            (au)                 -0.501309504112
   25    1    a          :            (au)                 -0.487039199597
   26    1    a          :            (au)                 -0.482126234433
   27    1    a          :            (au)                 -0.471371247668
   28    1    a          :            (au)                 -0.471261300342
   29    1    a          :            (au)                 -0.461178354029
   30    1    a          :            (au)                 -0.441495877124
   31    1    a          :            (au)                 -0.440970400187
   32    1    a          :            (au)                 -0.425046485467
   33    1    a          :            (au)                 -0.408323338380
   34    1    a          :            (au)                 -0.398745436066
   35    1    b          :            (au)                -89.485117559503
   36    1    b          :            (au)                -24.652425122502
   37    1    b          :            (au)                -24.651483995009
   38    1    b          :            (au)                -24.599830801413
   39    1    b          :            (au)                -24.598870131432
   40    1    b          :            (au)                -24.549408320879
   41    1    b          :            (au)                -24.545759958806
   42    1    b          :            (au)                 -8.858879503705
   43    1    b          :            (au)                 -6.810249962763
   44    1    b          :            (au)                 -6.809824925173
   45    1    b          :            (au)                 -6.809099426054
   46    1    b          :            (au)                 -1.577258863636
   47    1    b          :            (au)                 -1.397562858052
   48    1    b          :            (au)                 -1.376109566538
   49    1    b          :            (au)                 -1.350023932919
   50    1    b          :            (au)                 -1.285446927117
   51    1    b          :            (au)                 -1.256166943937
   52    1    b          :            (au)                 -0.845530770748
   53    1    b          :            (au)                 -0.713476056862
   54    1    b          :            (au)                 -0.696095800823
   55    1    b          :            (au)                 -0.673050360988
   56    1    b          :            (au)                 -0.552401055786
   57    1    b          :            (au)                 -0.543518908578
   58    1    b          :            (au)                 -0.528380362590
   59    1    b          :            (au)                 -0.501304775549
   60    1    b          :            (au)                 -0.487046640768
   61    1    b          :            (au)                 -0.482132893018
   62    1    b          :            (au)                 -0.471376733830
   63    1    b          :            (au)                 -0.471237137800
   64    1    b          :            (au)                 -0.461179836680
   65    1    b          :            (au)                 -0.441508862908
   66    1    b          :            (au)                 -0.440963889693
   67    1    b          :            (au)                 -0.425010663780
   68    1    b          :            (au)                 -0.408313948734
   69    1    b          :            (au)                 -0.398743445120
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -568.309155105170
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          8.366600e+00        -990.753191425126         -9.907532e+02
    1          5.481837e-01        -996.485155241966         -5.731964e+00
    2          2.060874e-01        -996.743463967525         -2.583087e-01
    3          1.169440e-01        -996.797967359672         -5.450339e-02
    4          6.512375e-02        -996.815968548943         -1.800119e-02
    5          7.881332e-02        -996.833012184689         -1.704364e-02
    6          2.248205e-02        -996.835108662726         -2.096478e-03
    7          2.456076e-02        -996.836609109950         -1.500447e-03
    8          9.059199e-03        -996.836809128795         -2.000188e-04
    9          1.066267e-02        -996.836959409110         -1.502803e-04
   10          4.766206e-03        -996.837010981512         -5.157240e-05
   11          4.976844e-03        -996.837047585383         -3.660387e-05
   12          1.786366e-03        -996.837057214692         -9.629309e-06
   13          1.659968e-03        -996.837062450936         -5.236245e-06
   14          1.210518e-03        -996.837065337303         -2.886367e-06
   15          6.526749e-04        -996.837066305939         -9.686362e-07
   16          3.057297e-04        -996.837066552344         -2.464051e-07
   17          1.910452e-04        -996.837066639397         -8.705251e-08
---------------------------------------------------------------------------
                      SCF converged in 17 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :        0.000000        0.000000        0.000000
    1       F            :        0.000000        0.000000        2.949295
    2       F            :       -2.085467        2.085467        0.000000
    3       F            :        2.085467       -2.085467        0.000000
    4       F            :        2.085467        2.085467        0.000000
    5       F            :       -2.085467       -2.085467        0.000000
    6       F            :        0.000000        0.000000       -2.949295
---------------------------------------------------------------------------
 Center of mass          :       -0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                994.676437675919
 E-N energy              :            (au)              -3507.561623848470
 Coulomb energy          :            (au)               1036.922147990783
 Exchange energy         :            (au)                -21.390791986667
 X-C energy              :            (au)                -66.672206618057
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                567.188970147096
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1564.026036786493
 Nuclear energy          :            (au)                567.188970147096
---------------------------------------------------------------------------
 Total energy            :            (au)             -9.968370666394e+02
                         :      (kcal/mol)             -6.255246983664e+05
                         :        (kJ/mol)             -2.617195339664e+06
                         :            (eV)             -2.712531725244e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -89.430363233636
    1    1    a          :            (au)                -24.838888970207
    2    1    a          :            (au)                -24.838885201969
    3    1    a          :            (au)                -24.838880663387
    4    1    a          :            (au)                -24.838879041095
    5    1    a          :            (au)                -24.838873134116
    6    1    a          :            (au)                -24.838871075490
    7    1    a          :            (au)                 -8.356198011378
    8    1    a          :            (au)                 -6.294722531122
    9    1    a          :            (au)                 -6.294722045622
   10    1    a          :            (au)                 -6.294721699193
   11    1    a          :            (au)                 -1.452632839340
   12    1    a          :            (au)                 -1.328286884726
   13    1    a          :            (au)                 -1.328286667985
   14    1    a          :            (au)                 -1.328283296864
   15    1    a          :            (au)                 -1.275657955802
   16    1    a          :            (au)                 -1.275655500006
   17    1    a          :            (au)                 -0.842283028229
   18    1    a          :            (au)                 -0.697124962824
   19    1    a          :            (au)                 -0.697124556838
   20    1    a          :            (au)                 -0.697122420711
   21    1    a          :            (au)                 -0.595988053989
   22    1    a          :            (au)                 -0.595986652378
   23    1    a          :            (au)                 -0.595986313764
   24    1    a          :            (au)                 -0.557466976173
   25    1    a          :            (au)                 -0.557464903756
   26    1    a          :            (au)                 -0.502159749402
   27    1    a          :            (au)                 -0.502159339060
   28    1    a          :            (au)                 -0.502158736423
   29    1    a          :            (au)                 -0.499795780091
   30    1    a          :            (au)                 -0.499794103665
   31    1    a          :            (au)                 -0.499793866125
   32    1    a          :            (au)                 -0.455902981007
   33    1    a          :            (au)                 -0.455901162007
   34    1    a          :            (au)                 -0.455900792710
   35    1    b          :            (au)                -89.430363232215
   36    1    b          :            (au)                -24.838889006692
   37    1    b          :            (au)                -24.838885357624
   38    1    b          :            (au)                -24.838880660521
   39    1    b          :            (au)                -24.838878952438
   40    1    b          :            (au)                -24.838873128125
   41    1    b          :            (au)                -24.838870988841
   42    1    b          :            (au)                 -8.356198010060
   43    1    b          :            (au)                 -6.294722530050
   44    1    b          :            (au)                 -6.294722043940
   45    1    b          :            (au)                 -6.294721698410
   46    1    b          :            (au)                 -1.452632837961
   47    1    b          :            (au)                 -1.328286881367
   48    1    b          :            (au)                 -1.328286673161
   49    1    b          :            (au)                 -1.328283295780
   50    1    b          :            (au)                 -1.275657954989
   51    1    b          :            (au)                 -1.275655504769
   52    1    b          :            (au)                 -0.842283023754
   53    1    b          :            (au)                 -0.697124964826
   54    1    b          :            (au)                 -0.697124553200
   55    1    b          :            (au)                 -0.697122421146
   56    1    b          :            (au)                 -0.595988054502
   57    1    b          :            (au)                 -0.595986651530
   58    1    b          :            (au)                 -0.595986313613
   59    1    b          :            (au)                 -0.557466975152
   60    1    b          :            (au)                 -0.557464906847
   61    1    b          :            (au)                 -0.502159751111
   62    1    b          :            (au)                 -0.502159338892
   63    1    b          :            (au)                 -0.502158736659
   64    1    b          :            (au)                 -0.499795780718
   65    1    b          :            (au)                 -0.499794104343
   66    1    b          :            (au)                 -0.499793865568
   67    1    b          :            (au)                 -0.455902982159
   68    1    b          :            (au)                 -0.455901161850
   69    1    b          :            (au)                 -0.455900793185
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -567.805846267092
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000000        0.000001        0.000027
 Magnitude               :            (au)                        0.000027
                         :         (Debye)                        0.000068
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000001        0.000027
 Magnitude               :            (au)                        0.000027
                         :         (Debye)                        0.000068
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  1h 53m 29s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
